website/docs/r/container_group.html.markdown

```diff
@@ -99,6 +99,8 @@ The following arguments are supported:
 
 * `key_vault_key_id` - (Optional) The Key Vault key URI for CMK encryption. Changing this forces a new resource to be created.
 
+* `key_vault_user_identity_id` - (Optional) The user assigned identity that has access to the Key Vault Key. If not specified, the RP principal named "Azure Container Instance Service" will be used instead. Make sure the identity has the proper `key_permissions` set, at least with `Get`, `UnwrapKey`, `WrapKey` and `GetRotationPolicy`.
```

Comment:
The ordering of the sentences here is a little confusing, are the `key_permissions` that are needed for the RP Principal or for `key_vault_user_identity_id`? Or both?

Reply:
It should be both identities.